plat/kvm/Config.uk

```diff
@@ -1,7 +1,7 @@
 menuconfig PLAT_KVM
        bool "KVM guest"
        default n
-       depends on (ARCH_X86_64 || (ARCH_ARM_64 && !HAVE_SYSCALL))
+       depends on ((ARCH_X86_64 && !HAVE_SMP) || (ARCH_ARM_64 && !HAVE_SMP && !HAVE_SYSCALL))
```

Comment:
I guess my branch might be a little outdated. Not sure why this changed but I did not edit this line and it is irrelevant to the GRUB ISO commit.

Reply:
That's from the SMP common API commits. Make sure that your branch is up-to-date with the staging branch.

Reply:
Okay fixed it...thanks!